### PR TITLE
update(readme): Update readme based on the recent csi changes

### PR DESCRIPTION
**Pull request by payes**
This PR updates the docs to:
1. Provision pools via cspc instead of spc.
2. Expand cStor volumes using OpenEBS CSI driver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,15 +68,14 @@ openebs-csi-node-56t5g     2/2     Running   0          6m13s
...
+   that the specified pools list should be greater than or equal to
+   the number of replicas required for the volume.
 
-   The following command will create the specified number of cStor Pools
-   using the Sparse files. 
+   The following command will create the specified cStor Pools in the cspc yaml:
 
    ```
-   kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/spc.yaml
+   kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/cspc.yaml
```
Comment (kmova) [inline README.md]: this file needs to be added.

File: README.md [hunk 2]
```diff
@@ -16,12 +16,13 @@ Before setting up OpenEBS CSI driver make sure your Kubernetes Cluster
...
 1. You will need to have Kubernetes version 1.14 or higher
-2. You will need to have OpenEBS Version 1.1 or higher installed. 
+2. You will need to have OpenEBS Version 1.2 or higher installed. 
    The steps to install OpenEBS are [here](https://docs.openebs.io/docs/next/quickstart.html)
 3. iSCSI initiator utils installed on all the worker nodes
 4. You have access to install RBAC components into kube-system namespace.
    The OpenEBS CSI driver components are installed in kube-system 
-   namespace to allow them to be flagged as system critical components. 
+   namespace to allow them to be flagged as system critical components.
+5. You will need to turn on  ExpandCSIVolumes and ExpandInUsePersistentVolumes feature gates on  kubelets and kube-apiserver:
```
Comment (kmova) [inline README.md]: Are there GA/Beta in 1.15 K8s?

Comment (payes) [reply README.md]: ExpandInUsePersistentVolumes has been moved to Beta while ExpandCSIVolumes is still in alpha stage as of 1.15.

File: README.md (hunk 1 above)
Comment (payes) [reply README.md]: This file has been added in a separate PR https://github.com/openebs/csi/pull/34

File: README.md [hunk 3]
```diff
@@ -175,3 +176,18 @@ in progress, the application pod may throw some errors like:
 
 On successful completion of the above steps the application pod can 
 be seen in running state.
+
+### Expand a cStor volume using OpenEBS CSI driver
+
+#### Notes:
+- Only dynamically provisioned volumes can be resized.
+- You can only resize volumes containing a file system if the file system is XFS, Ext3, or Ext4.
+- Make sure that the storage class has the allowVolumeExpansion field set to true when the volume is provisioned.
```
Comment (prateekpandey14) [inline README.md]: nit: add quotes `allowVolumeExpansion: true` ?

File: README.md (hunk 1 above)
Comment (kmova) [reply README.md]: I don't see cspc.yaml yet in that PR.

Comment (payes) [reply README.md]: added it now, had missed pushing the file.

**Pull request merged**